RELEASE_NOTES.md

```diff
@@ -6,6 +6,7 @@
 - fixed: `git machete` now correctly displays only available commands (without help topics)
 - fixed: `git machete traverse -y -M` no longer exits with error
 - fixed: `git machete` commands no longer exit with stack trace when interrupted by Ctrl+C
+- improved: instructions on how to install `git-machete` locally for development purposes
```

Comment:
Not worth mentioning in release notes. In general, let's not include stuff that's not relevant for everyday users (excl. developers/contributors)

Reply:
I sometimes put stuff like `fixed: Homebrew release process` in 3.12.4 but mostly coz nothing else happened in that release, just to put anything not to break the scripts xD